draft-ietf-mmusic-ice-sip-sdp-36.xml

```diff
@@ -436,6 +428,13 @@ Also as described in <xref target="RFC8445"/>, once the controlling agent has no
 a candidate pair for a data stream, the agent MUST NOT nominate another pair
 for that data stream during the lifetime of the ICE session (i.e. until
 ICE is restarted).
+</t>
+<t>
+  <xref target="draft-holmberg-ice-pac"/> provides a mechanism for
+  allowing the ICE process to run long enough in order to find working candidate pairs,
+  by waiting for potential peer-reflexive candidates, eventhough all current candidate pairs
+  associated with a check list have either failed or been discarded. It is OPTIONAL
+  for an ICE agent to support the mechanism.
```

Comment:
Do we need to add something here that ice-pac also applies when remote party did not send any candidates, not only when all candidates are discarded?

I think something like "even though *no candidate pairs were received from the remote agent or* all current candidate pairs  associated with a check list have either failed or been discarded" would be more accurate.

Reply:
Sounds good. I will fix that.

Reply:
I have updated the pull request based on Roman's comment. In addition, I updated the ICE PAC reference.